### PR TITLE
Remove validation check for boot disk type in nodeset module

### PR DESCRIPTION
**Pull request by harshthakkar01**
This PR removes the validation check in the boot disk type in nodeset module and updates description for supported disk type.

For manual testing, I tested this with c3-standard-22 machine for hyperdisk-balanced and c3-standard-88 machine with disk 70 G for hyperdisk-extreme disk type. Added result in the canonical bug.

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf [hunk 1]
```diff
@@ -117,14 +117,9 @@ variable "tags" {
 }
 
 variable "disk_type" {
-  description = "Boot disk type, can be either pd-ssd, pd-standard, pd-balanced, or pd-extreme."
+  description = "Boot disk type, can be either hyperdisk-balanced, hyperdisk-extreme, pd-ssd, pd-standard, pd-balanced, or pd-extreme."
```
Comment (nick-stroud) [inline community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf:120]: From side discussion, you mentioned you tested `hyperdisk-balanced`. Please add description of manual testing to PR description. Did you also try deploying with `hyperdisk-extreme`?

Comment (nick-stroud) [inline community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf:120]: Can you make these same changes to login and controller?

**Review by nick-stroud — CHANGES_REQUESTED** [review]

File: community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf (hunk 1 above)
Comment (harshthakkar01) [reply community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf:120]: Updated the desc.

Comment (harshthakkar01) [reply community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf:120]: Done.

**Review by nick-stroud — APPROVED** [review]

**Pull request merged**